src/app/page.tsx

```diff
@@ -32,27 +38,32 @@ const Home = (): JSX.Element => {
     },
   ];
 
-  // TODO Remove this mockup data
-  const cards: JSX.Element[] = [Card({ ...data[0] }), Card({ ...data[1] }), Card({ ...data[2] })];
-
-  // TODO Remove this mockup data
-  function repeat(cards: JSX.Element[], repetition: number = 10): JSX.Element[] {
-    const items: JSX.Element[] = [];
-    for (let i: number = 0; i < repetition; ++i)
-      items.push(cards[Math.floor(Math.random() * cards.length)]);
-
-    return items;
-  }
+  const [showAlert, setShowAlert] = useState(false);
 
   return (
-    <div className="flex flex-col mt-[72px] pt-4 px-4">
-      {/* TODO Remove this mockup data */}
-      <Card {...data[0]} />
-      <Card {...data[1]} />
-      <Card {...data[2]} />
-      {repeat(cards, 7)}
-      <CreateButton />
-    </div>
+    <AlertContext.Provider value={{ showAlert, setShowAlert }}>
+      <Alert />
+
+      <div
+        className={clsx(
+          'flex flex-col pt-4 px-4',
+          showAlert && 'mt-[112px]',
+          !showAlert && 'mt-[72px]'
```

Comment:
```suggestion
          showAlert ? 'mt-[112px]' : 'mt-[72px]',
```

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/7/commits/7e3a825fb939daf367835007e1506ec2381dffcb, thank you